### PR TITLE
JanusGraph release 1.1.0 [cql-tests] [tp-tests]

### PR DESCRIPTION
**Pull request by porunov**
-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered?

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -154,7 +154,36 @@ To disable the `drop()` step optimization and maintain the previous behavior, us
...
+* textNotPrefix
+* textNotRegex
+
+##### Vertex mutation optimizations
+
+The following improvements are made for adding new vertex or updating vertex properties:
+
+* Improved operations to detect vertex changes during transaction-commit from O(N) to O(1);
+* Optimizing properties search by key for newly-created vertex;
+* Optimizing previous property/edge search during vertex update;
```
Comment (cdegroc) [inline docs/changelog.md:184]: **❓ question:** ‏Is there one or more PRs associated with those optimizations we could reference? If not in the doc itself, maybe here as a PR comment?

Comment (porunov) [reply docs/changelog.md:184]: The PR is: https://github.com/JanusGraph/janusgraph/pull/4292
I just added it as a second commit to not trigger full CI

Comment (porunov) [reply docs/changelog.md:184]: Seems CI is triggered anyways. I guess it's per PR not per commit. Anyways, I will squash all commits when merging them into the `master` branch

File: docs/changelog.md [hunk 2]
```diff
@@ -178,10 +209,10 @@ compile "org.janusgraph:janusgraph-core:1.0.1"
 * Apache HBase 2.5.8
```
Comment (cdegroc) [inline docs/changelog.md:209]: **❓ question:** ‏I think that should be 2.6.0 based on [this line](https://github.com/porunov/janusgraph/blob/release/v1.1.0/pom.xml#L71)

File: docs/changelog.md [hunk 3]
```diff
@@ -178,10 +209,10 @@ compile "org.janusgraph:janusgraph-core:1.0.1"
...
 * Oracle BerkeleyJE 7.5.11
 * ScyllaDB 5.1.4
```
Comment (cdegroc) [inline docs/changelog.md:211]: **❓ question:** ‏Should it be 6.2.0? (based on [this line](https://github.com/porunov/janusgraph/blob/release/v1.1.0/pom.xml#L107))

Comment (porunov) [reply docs/changelog.md:211]: That is for Version 1.0.1 and not 1.1.0. We didn't merge it to `v1.0` branch.

File: docs/changelog.md (hunk 2 above)
Comment (porunov) [reply docs/changelog.md:209]: Same, that is for Version 1.0.1 and not 1.1.0. Several lines above description for 1.1.0 contains `2.6.0` as you specified

Comment (cdegroc) [reply docs/changelog.md:209]: Sorry about that!

**Review by cdegroc — APPROVED** [review]
Thanks 🙏

File: docs/changelog.md (hunk 2 above)
Comment (porunov) [reply docs/changelog.md:209]: No worries. Thank you!
I will be merging this PR only after 1.0.1 PR #4722 
So that dates are relevant

**Pull request merged**